spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -123,7 +124,263 @@ In Jakarta Data, an entity refers to a fundamental data representation and manag
+    private LocalDate expirationDate;
+}
+
+public class Citizen {
+    private Long id;
+    private String name;
+    private Passport passport; // One-to-One relationship with Passport for relational database
+}
+----
+
+Here are some possible representations of the `Citizen` and `Passport` entities in table and JSON formats:
+
+.Citizen Table:
+[options="header"]
+|===
+| id (Primary Key) | name | passport_id (Foreign Key)
+| 1                | John Doe | 1
+|===
+
+.Passport Table:
+[options="header"]
+|===
+| id (Primary Key) | passportNumber | expirationDate
+| 1                | A123456        | 2023-12-31
+|===
+
+These tables represent the data from the JSON structure you provided earlier.
+In a relational database, the `Citizen` and `Passport` entities are stored in separate tables, connected by a foreign key relationship.
+
+[source,json]
+----
+{
+  "id": 1,
+  "name": "John Doe",
+  "passport": {
+    "passportNumber": "A123456",
+    "expirationDate": "2023-12-31"
+  }
+}
+----
+
+Entities in a domain model often have relationships with other entities. In some cases, an entity may have a collection of another type of entity, creating a one-to-many or many-to-many relationship. This scenario explores how such relationships are represented and managed in the context of Jakarta Data.
+
+Consider the example of an `Author` entity associated with multiple `Book` entities. This relationship allows an author to be linked to multiple books they have authored. While this structure remains relatively unchanged for NoSQL databases, it introduces specific considerations in relational databases, where it typically generates auxiliary tables to manage the relationship. We'll explore these representations in both JSON and relational database formats.
+
+[source,java]
+----
+// Entity
+public class Author {
+
+    private UUID id;
+
+    private String name;
+
+    private List<Book> books;
+}
+
+// If in a relational database, Book might also be an entity
+public class Book {
+
+    // This field might not be required for some NoSQL database modeling
+    private Long id;
+
+    private String title;
+
+    private String category;
+
+    // Other fields and methods
+}
+----
+
+
+[source,json]
+----
+{
+  "id": "6f6d665d-5585-46cd-8b9b-61a559de0e13",
+  "name": "John Doe",
+  "books": [
+    {
+      "title": "Sample Book 1",
+      "category": "Fiction"
+    },
+    {
+      "title": "Sample Book 2",
+      "category": "Non-fiction"
+    }
+  ]
+}
+----
+
+In the JSON representation, an `Author` entity can be associated with multiple `Book` entities within an array.
+
+Certainly, here are the tables with content based on the JSON data structure:
+
+.Author Table:
+[options="header"]
+|===
+| id (Primary Key) | name
+| 1                | John Doe
+| 2                | Jane Smith
+|===
+
+.Book Table:
+[options="header"]
+|===
+| id (Primary Key) | title | category
+| 1                | Book A | Fiction
+| 2                | Book B | Non-fiction
```

Comment:
Data values should be the same in both JSON and table examples given that we previously stated the table contents are based on the JSON data structure,
```suggestion
| 1                | Sample Book 1 | Fiction
| 2                | Sample Book 2 | Non-fiction
```